### DISCUSSION TITLE
Data-include attribute tag for the exclusions script extension

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1683] [Anything Else]
Currently, it's possible to use the [exclusions script extension](https://plausible.io/docs/excluding-pages) to exclude certain pages from being tracked.

Would be great to also have the option to only include certain pages. This could work with the same script extension and a `data-include` attribute tag.

Use case:

My site is built on subfolders, indicating the language of the page like `mysite.com/en/...` and `mysite.com/es/...`. I want their stats as separate dashboards in Plausible. My site-building tool does not support inserting a different snippet on different subfolders.

**Comment by metmarkosaric** [comment]
Data-include is now available! See https://plausible.io/docs/excluding-pages
